euphonic/qpoint_frequencies.py

```diff
@@ -107,6 +110,12 @@ def calculate_dos(self, dos_bins: Quantity,
             Scalar float Quantity in energy units. Sets a lower limit on
             the mode widths, as mode widths of zero will result in
             infinitely sharp peaks
+        adaptive_method
+            String. Specifies whether to use slow, reference adaptive method or
+            faster, approximate method. Allowed options are 'default', or 'fast'.
+        error
+            Scalar float. Acceptable percentage error for gaussian approximations when using
+            the fast adaptive method.
```

Comment:
Is this actually "percentage" (i.e. 0.001% = 0.00001)? I expect we can give a stricter definition (e.g. RMS error/total intensity).

Reply:
Yeah, I can see that it is a bit unclear what percentage refers to - I will add in a stricter definition